blogposts/announcing-sourcegraph-3.0.md

```diff
@@ -0,0 +1,243 @@
+
+Previously, Sourcegraph implemented these HTTP features internally. We found that most admins preferred to wrap Sourcegraph with Nginx (sometimes using [ingress-nginx](https://github.com/kubernetes/ingress-nginx)) anyway, so we decided to make that the preferred and only way. Now that Sourcegraph relies on Nginx, you have the full power and configurability of Nginx at your disposal.
+
+### Faster page load times
+
+We trimmed the initial bundle size by ~50% for the Sourcegraph web app. This speeds up page loads on Sourcegraph.com and your own Sourcegraph instance. Thanks, [Webpack](https://webpack.js.org/)! (As a [JS Foundation member](https://js.foundation/about/members), Sourcegraph is proud to support Webpack.)
+
+### Datadog metrics extension
+
+The new [Datadog metrics extension](https://sourcegraph.com/extensions/sourcegraph/datadog-metrics) adds **View metric** links at the end of lines of code that contain calls to record [Datadog](https://www.datadoghq.com)/statsd metrics. This makes it easy to see how code is behaving *in production and in real time* when you're reading and reviewing code.
+
+Because this is a [Sourcegraph extension](https://docs.sourcegraph.com/extensions), it works on Sourcegraph and on your code host (such as on GitHub code files and PRs).
+
+For example, if your code records the time it takes to respond to queries, this extension makes it easy to jump to the query response time statistics from your code:
+
+![Line of code with Datadog metrics link](./announcing-sourcegraph-3.0/datadog_metrics.png)
+
+## Documentation
+
+The [main docs.sourcegraph.com page](https://docs.sourcegraph.com/) was reorganized to make the most common documentation easier to get to. We've also added a lot more documentation pages throughout.
+
+## Engineering
+
+This release includes more solid foundations for several Sourcegraph features:
+
+- [Sourcegraph extensions](https://docs.sourcegraph.com/extensions) are the sole source of code intelligence now. We removed the old way, which required a lot more Sourcegraph-specific custom integration work to add or configure a new language.
+- The [management console](https://docs.sourcegraph.com/admin/management_console) makes it so that all configuration can be edited online and in-band, which means future Sourcegraph releases can auto-migrate their own config instead of requiring manual (and error-prone) steps by the site admin.
+- The new concept of [external service connections](https://docs.sourcegraph.com/admin/repo/add) will allow us to improve Sourcegraph's synchronization of repositories in a future release.
+- Standardizing on Nginx for serving HTTP lets us remove a lot of complex core code to support numerous use cases better handled by Nginx.
+
+## Thank you
+
+Thank you to the many people who contributed to make Sourcegraph better since our last release!
+
+- [@sabyasachi](https://github.com/sabyasachi)
```

Comment:
@ryan-blunden 	Could you please do your magic and update this list to add contributors between 3.0 beta and now?

Reply:
On it!